roles/jd-server/README.md

```diff
@@ -0,0 +1,27 @@
+# JD Server
+
+The JD Server is a Sv2 proxy, It is also the entity responsible for Pool-side block propagation in case of valid blocks found by miners connected to the pool (who are using the Job Declaration Protocol).
```

Comment:
I don't think that Jds server is a proxy, because there is communitation with the pool. It is also reponsible for providing signed tokens. This tokens then are enclosed in each share that is sent to the pool
@Fi3  can you confirm that?